controller/src/beerocks/master/controller_ucc_listener.cpp

```diff
@@ -80,6 +81,7 @@ bool controller_ucc_listener::send_cmdu_to_destination(Socket *sd,
 bool controller_ucc_listener::handle_dev_set_config(
     std::unordered_map<std::string, std::string> &params, std::string &err_string)
 {
+    std::unordered_set<std::string> al_mac_cleared_conf;
```

Comment:
I would have used sMacAddr instead of std::string, but that's probably nitpicking.

Hang on, it's not nitpicking at all: you need the first commit because you're using a string here instead of sMacAddr! This shows that there is a good reason to keep both the upper and lower case in the test.